python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -515,7 +515,7 @@ class NearestNeighbors(Base,
             When set to True, the kneighbors method will automatically
             convert the inputs to np.float32.
 
-        _output_cumlarray : bool, optional (default = False)
+        _output_type : bool, optional (default = False)
```

Comment:
This name change seems less clear than the previous one since this is a boolean value as opposed to a list of options